app/models/specs/models/like_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  it 'tests that Like model is created correctly' do
+    like = Like.new
+    like.build_user(name: 'Lolade')
+    like.build_post(text: 'new post 2')
+    expect(like).to be_valid
+  end
```

Comment:
- [ ] Hello and excellent work, but the spec files should be created using the Rpsec.